README.md

```diff
@@ -144,13 +144,23 @@ Parameter | Description | Default
 `ingress.tls` | Boolean that controls whether TLS should be enabled for the ingress | `true`
 `ingress.secretName` | Name of the secret that holds the TLS certificates, the secret must be manuallly created unless cert-manager is installed | `""`
 `ingress.issuer` | Name of the issuer to use when requesting the TLS certifiactes, requires that `cert-manager` is installed and configured | `""`
+`filedatabase_host` | External filedatabase host | `""`
+`filedatabase_port` | External filedatabase port | `9059`
+`keys_host` | External keys host | `""`
+`keys_port` | External keys port | `9095`
+`dataedge_host` | External dataedge host | `""`
+`dataedge_port` | External dataedge port | `9050`
+`res_host` | External res host | `""`
+`res_port` | External res port | `9090`
+`dataedge.deploy` | Set to false if using a dataedge service | `true`
 `dataedge.replicaCount` | desired number of replicas | `1`
 `dataedge.repository` | dataedge container image repository | `cscfi/ega-dataedge`
 `dataedge.imageTag` | dataedge container image version | `"latest"`
 `dataedge.imagePullPolicy` | dataedge container image pull policy | `IfNotPresent`
 `dataedge.keystorePass` | keystore password | `dataedge`
 `dataedge.port` | dataedge container port | `8080`
 `dataedge.servicePort` | dataedge service port | `9059`
+`filedatabase.deploy` | Set to false if using a filedatabase service | `true`
```

Comment:
Shouldn't it be `if not using` rather than `if using` ?

Reply:
true, will correct for the others as well